test/environments.js

```diff
@@ -137,10 +137,12 @@ class ShardedEnvironment extends EnvironmentBase {
     this.host = 'localhost';
     this.port = 51000;
 
+    // Disabled only connecting to a single shard for now
```

Comment:
Can you add a brief description of why or a `TODO`?

Reply:
will do